braindecode/datasets/base.py

```diff
@@ -109,34 +109,37 @@ def __init__(self, list_of_ds):
         super().__init__(list_of_ds)
         self.description = pd.DataFrame([ds.description for ds in list_of_ds])
 
-    def split(self, some_property=None, split_ids=None):
-        """Split the dataset based on some property listed in its description
+    def split(self, by):
```

Comment:
technically you are breaking public API here again. Maybe we want to reach a proper version
before stabilizing API. I feel there are still many moving parts.

Reply:
My bad, I am not yet used to deprecating and not breaking the API. And yes, a lot of stuff is moving.

Reply:
@gemeinl if you care about the design of the API it's great. Don't worry for now about breaking API as long as it's for the best.